tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Copyright 2022 Canonical Ltd.
+# Licensed under the GPLv3, see LICENCE file for details.
```

Comment:
This license headers is not the one we should be using. Use
```
# Copyright 2022 Canonical Ltd.
# See LICENSE file for licensing details.
```
instead.
I assume you got it from another file. Can you please change all occurrences?

Reply:
I have changed all occurrences.
I have also notice the python file `ingress.py` under `lib/charms/nginx_ingress_integrator/v0` does not start with any licensing info. Is this fine?

Reply:
It might be. Since those libraries are downloaded from charmhub we'd have to update the appropriate sources.
Thank you

Reply:
https://github.com/canonical/nginx-ingress-integrator-operator/pull/37 to fix the lack of copyright notice in that library file.